schemas/itemset/2.0/schema.json

```diff
@@ -22,7 +22,8 @@
             "type": "number"
         },
         "type": {
-            "type": "string"
+            "type": "string",
```

Comment:
Add enum with `materialised` otherwise, the system accepts any string.

Reply:
done